systemtests/src/main/java/io/enmasse/systemtest/clients/ClientUtils.java

```diff
@@ -145,4 +150,29 @@ public void receiveMessages(AmqpClient amqpClient, String address, int count) th
         assertThat("Incorrect count of messages received from " + address + ". Got " + receiverStatus.getNumReceived(),
                 receiverStatus.getResult().get(timeoutMs, TimeUnit.MILLISECONDS).size(), is(count));
     }
+
+    public static void preparePolicyClients(AbstractClient sender, AbstractClient receiver, Address dest, AddressSpace addressSpace) throws Exception {
+        ClientArgumentMap arguments = new ClientArgumentMap();
+        UserCredentials credentials = new UserCredentials("test", "test");
+
+        arguments.put(ClientArgument.USERNAME, credentials.getUsername());
+        arguments.put(ClientArgument.PASSWORD, credentials.getPassword());
+        arguments.put(ClientArgument.BROKER, getInnerMessagingRoute(addressSpace).toString());
+        arguments.put(ClientArgument.ADDRESS, dest.getSpec().getAddress());
+        arguments.put(ClientArgument.COUNT, "5");
+        arguments.put(ClientArgument.MSG_CONTENT, "msg no. %d");
+        arguments.put(ClientArgument.TIMEOUT, "30");
+        arguments.put(ClientArgument.CONN_SSL, "true");
+        arguments.put(ClientArgument.LOG_MESSAGES, "json");
+
+        sender.setArguments(arguments);
+        arguments.remove(ClientArgument.MSG_CONTENT);
+        receiver.setArguments(arguments);
+    }
+
+    private static Endpoint getInnerMessagingRoute(AddressSpace addressSpace) throws Exception {
+        return new Endpoint(String.format("%s-%s.%s.svc.cluster.local",
```

Comment:
This should use the addressSpace.getStatus().getEndpointStatuses(name=="messaging").getServiceHost() instead of relying on the annotation.